src/input/Input.js

```diff
@@ -115,14 +114,18 @@ class Input extends Component {
   }
 }
 
+const elementOrObject = PropTypes.oneOfType([
```

Comment:
Make we can make this a helper too? I think we use it pretty often. Duplicated in ListItem as well. https://github.com/react-native-training/react-native-elements/blob/master/src/list/ListItem.js#L268

Reply:
Yea sure, good point